gradle.properties

```diff
@@ -1,3 +1,15 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx4096m
\ No newline at end of file
+# Less than 2G definitely slows things down.
+org.gradle.jvmargs=-Xmx2G -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
Previously we needed more than 2g for running all the tests, but maybe memory requirements did change in the meantime?

Enabling heap-dump always can lead to leftover files and disk filling up in CI, can we at least send it somewhere else, e.g. /tmp?

Reply:
> Previously we needed more than 2g for running all the tests, but maybe memory requirements did change in the meantime?

I was able to run everything with Xmx=1GB, but it got slower. I found 2GB the optimal setting.
Also bear in mind, a lots of old Laptop Computers still are equipped with 4GB only (less 1GB for shared graphics making it 3 GB effectively). I think POI still should build on such a Laptop and the original build froze the system on my junk laptop computer.

> Enabling heap-dump always can lead to leftover files and disk filling up in CI, can we at least send it somewhere else, e.g. /tmp?

I will just remove it. This has no direct advantage. Apologies for accidentally "re-introducing" such stuff, I was not aware that there was a history on this already.

Reply:
Thanks, if the full build via the "jenkins" task still works it should be fine.